packages/wallet/backend/src/gatehub/client.ts

```diff
@@ -58,61 +65,70 @@ export class GateHubClient {
     return `https://onboarding.${this.mainUrl}`
   }
 
-  async getWithdrawalUrl(): Promise<string> {
+  async getWithdrawalUrl(managerUserId: string): Promise<string> {
     const token = await this.getIframeAuthorizationToken(
       this.clientIds.onOffRamp,
-      DEFAULT_APP_SCOPE
+      DEFAULT_APP_SCOPE,
+      managerUserId
     )
 
     return `${this.rampUrl}/?paymentType=${PAYMENT_TYPE.withdrawal}&bearer=${token}`
   }
 
-  async getDepositUrl(): Promise<string> {
+  async getDepositUrl(managerUserId: string): Promise<string> {
     const token = await this.getIframeAuthorizationToken(
       this.clientIds.onOffRamp,
-      DEFAULT_APP_SCOPE
+      DEFAULT_APP_SCOPE,
+      managerUserId
     )
 
     return `${this.rampUrl}/?paymentType=${PAYMENT_TYPE.deposit}&bearer=${token}`
   }
 
-  async getOnboardingUrl(): Promise<string> {
+  async getOnboardingUrl(managerUserId: string): Promise<string> {
```

Comment:
There are multiple places where the variable is named `managerUserId`. I'll try to point out all of them:
```suggestion
  async getOnboardingUrl(managedUserId: string): Promise<string> {
```